### ISSUE TITLE
modification

### ISSUE THREAD

**Issue by ipk1** [hakluke/hakrevdns#12]
hey,
is there a way that the last "period" when checking can be omitted in the results? currently using sed to remove it.
3.0.84.109     twofasapi.paypal.com.
173.0.84.107     cupapiserv.paypal.com.
173.0.84.113     mpltapi.paypal.com.
173.0.84.116     cld-edge-origin-api.paypal.com.
173.0.84.114     www-carrier.paypal.com.
173.0.84.117     api-aa-3t.paypal.com.
173.0.84.120     trinityadminauth.paypal.com.
173.0.84.115     phy-edge-origin-api.paypal.com.
173.0.84.119     cors.api.paypal.com.
173.0.84.118     appmanagement.paypal.com.
173.0.84.122     proxo.paypal.com.
173.0.84.121     ca-webapi-vip-

thanks
ipk1

**Comment by hakluke** [comment]
Hey ipk1!

All domains are still totally valid with the . on the end, but it shouldn't be too hard to remove them :)

**Comment by pczajkowski** [comment]
It's already being done if called with Domain option `if opts.Domain { fmt.Println(strings.TrimRight(a, ".")) }`, so it may be worth changing the other case for consistency sake?

**Comment by hakluke** [comment]
Aaaah I see, yeah that's a bug.

**Comment by hackerpain** [comment]
> is there a way that the last "period" when checking can be omitted in the results? currently using sed to remove it.

Yes! I was facing the same issue, so I implemented the ```-d``` flag to trim the trailing ```.```, so that its easier to pass the output for further processing.
